backend/DB/models/jwt.cjs

```diff
@@ -0,0 +1,27 @@
+"use strict";
+const { Model } = require("sequelize");
+module.exports = (sequelize, DataTypes) => {
+  class JWT extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    static associate(models) {
+      // define association here
```

Comment:
I'm not very strong with sequelize, but think we should add an association for jwts belonging to users. See https://github.com/JustUtahCoders/flax-crm/blob/173715eba031c44358b61ebd385b60f0af593ac5/backend/DB/models/field.cjs#L12 as an example

Reply:
Yes!  I did this: `      models.JWT.belongsTo(models.User);`